src/Aspire.Hosting/ApplicationModel/EndpointAnnotation.cs

```diff
@@ -31,7 +31,8 @@ public sealed class EndpointAnnotation : IResourceAnnotation
     /// <param name="targetPort">This is the port the resource is listening on. If the endpoint is used for the container, it is the container port.</param>
     /// <param name="isExternal">Indicates that this endpoint should be exposed externally at publish time.</param>
     /// <param name="isProxied">Specifies if the endpoint will be proxied by DCP. Defaults to true.</param>
-    public EndpointAnnotation(ProtocolType protocol, string? uriScheme = null, string? transport = null, [EndpointName] string? name = null, int? port = null, int? targetPort = null, bool? isExternal = null, bool isProxied = true)
+    /// <param name="displayProperties">Function to configure Aspire Dashboard display properties for this endpoint.</param>
+    public EndpointAnnotation(ProtocolType protocol, string? uriScheme = null, string? transport = null, [EndpointName] string? name = null, int? port = null, int? targetPort = null, bool? isExternal = null, bool isProxied = true, Action<EndpointDisplayProperties>? displayProperties = null)
```

Comment:
undo this Its a breaking change.

Reply:
There's no real need to set this in the constructor. Callers can just update the property after construction.